### PR TITLE
Feature/update dockerfile

### PR DESCRIPTION
**Pull request by garyluu**
A few changes:
- update node to latest lts
- update dredd to latest
- update TRS version to test to 2.0.0
- use HTML report (it now has the summary and equally informative as the CLI output)

### CODE REVIEW

**Comment by denis-yuen** [comment]
FYI, @NatalieEO  this will affect https://github.com/dockstore/dockstore/issues/2838

**Comment by garyluu** [comment]
I think this might actually be on the live server already

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
-FROM node:12.10-slim
+FROM node:lts-slim
```
Comment (denis-yuen) [inline Dockerfile:1]: What does this mean in terms of version? 
Will this automatically update to the latest LTS version or is LTS-slim a specific LTS version?

Comment (garyluu) [reply Dockerfile:1]: It should automatically update.

Comment (denis-yuen) [reply Dockerfile:1]: In that, case could you rig up this Dockerfile to a cron build https://docs.travis-ci.com/user/cron-jobs/ , run maybe weekly so we have some heads-up/record when/if this breaks.

**Review by denis-yuen — APPROVED** [review]

**Review by NatalieEO — APPROVED** [review]

**Comment by garyluu** [comment]
Can't seem to test this with cron, but it should work.  I've tested it with on push though: https://github.com/ga4gh/tool-registry-validator/runs/475284177?check_suite_focus=true

**Pull request merged**